compiler/modules/front.syntax/astString.bal

```diff
@@ -205,14 +205,15 @@ function stmtToWords(Word[] w, Stmt stmt) {
     else if stmt is BreakContinueStmt {
         w.push(stmt.breakContinue, ";");
     }
-    else if stmt is CheckingStmt {
-        w.push(stmt.checkingKeyword);
-        exprToWords(w, stmt.operand);
-        w.push(";");
-    }
     else {
-        // This deals with function call and method call
-        exprToWords(w, stmt);
+        CallExpr expr = stmt.expr;
+        if expr is CheckingCallExpr {
```

Comment:
Why do you need to handle CheckingCallExpr separately here?

Reply:
removed (passed directly to exprTo words).